.husky/pre-commit

```diff
@@ -5,19 +5,19 @@
 cd "$(dirname -- "$0")/.." || exit 1
 
 # Ensure node is available (same as asciinema-rec.sh)
-if [[ ! -x "$(command -v node)" ]]; then
-  if [[ ! -x "$(command -v nvm)" && -f ~/.nvm/nvm.sh ]]; then
+if [ ! -x "$(command -v node)" ]; then
+  if [ ! -x "$(command -v nvm)" ] && [ -f ~/.nvm/nvm.sh ]; then
```

Comment:
To thread on a random diff line:

@dcramer what's your OS and your default shell? I should probably add a ShellCheck CI check to catch this kind of syntax.

Reply:
this is Ubuntu on WSL2 (though I run a current gen Macbook as well - should work there too afaik)

Reply:
Here's some proof that ShellCheck also wants me to make these changes: https://github.com/emmercm/igir/actions/runs/5909212635/job/16029534496?pr=580